apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/InstanceConfiguration.tsx

```diff
@@ -71,28 +82,46 @@ const InstanceConfigurationUI = () => {
 
   const nodes = useMemo(
     () =>
-      isSuccess
-        ? generateNodes(primary, replicas, {
+      isSuccessLoadBalancers && isSuccessLoadBalancers
```

Comment:
Twice, just to be sure its really loaded? :)

Reply:
oh LOL good catch 🙏 🙏